tests/conftest.py

```diff
@@ -207,12 +208,24 @@ def verify_squad_owners(items):
                     "Ignoring test case %s as it has a marker in the ignore list",
                     item.name,
                 )
+
+            # Verify tests are decorated with the correct squad owner
             elif not any(["_squad" in marker for marker in item_markers]):
                 log.debug("%s is missing a squad owner marker", item.name)
                 items_without_squad_marker.update({item.name: item.fspath.strpath})
 
+            # Verify red squad tests are decorated with either @mcg or @rgw
+            elif (
+                "red_squad" in item_markers
+                and "mcg" not in item_markers
+                and "rgw" not in item_markers
+            ):
```

Comment:
```suggestion
            elif "red_squad" in item_markers and not any(mark in item_markers for mark in ["mcg", "rgw"]):
```

Reply:
I would not block this big PR when we have that many approvals needed by this change and loose the all approvals, suggest to do this change if insist on next PR.